docs/troubleshooting/authentication/custom_authentication_in_flutterFlow.md

```diff
@@ -0,0 +1,79 @@
+---
+keywords: ['authentication', 'api', 'configuration']
+slug: /custom-authentication-in-flutterflow
+title: Custom Authentication in FlutterFlow
+---
+# Custom Authentication in FlutterFlow
+
+**Pre-Requisites to Enabling Custom Authentication:**
```

Comment:
Check docs guidelines on how to write a pre-requisite in our doc - https://www.notion.so/flutterflow/Docs-Contribution-Guidelines-Shared-Externally-42829c305df441f6b061b4095abac8a8#32ed54d0e6804385b4773a0e61dda14b